src/current/v22.2/aws-dms.md

```diff
@@ -203,8 +205,8 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 
 - For visibility into migration problems:
 
-    - Check the `SQL_EXEC` [logging channel](logging-overview.html#logging-channels) for log messages related to `COPY` statements and the tables you are migrating.
-    - Check the [Amazon CloudWatch logs that you configured](#step-2-2-task-settings) for messages containing `SQL_ERROR`.
+    - Check the [Amazon CloudWatch logs that you enabled](#step-2-2-task-settings) for messages containing `SQL_ERROR`.
+    - Check the CockroachDB logs, especially the `SQL_EXEC` and `DEV` [logging channels]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels), for messages related to `COPY` statements and the tables you are migrating. To access CockroachDB {{ site.data.products.dedicated }} logs, you should have configured log export to Amazon CloudWatch [before beginning the DMS migration](#before-you-begin).
```

Comment:
If im not mistaken, log export doesnt ship the DEV channel logs which are sometimes important. We only support SESSIONS, OPS, HEALTH, STORAGE, SQL_SCHEMA, USER_ADMIN, PRIVILEGES, SENSITIVE_ACCESS, SQL_EXEC, and SQL_PERF in this manner.

We should also call out that if you don't choose to redact the logs, PII may go to cloudwatch. But if you redact, you might not know the data that caused the issue fully.

Reply:
Updated - PTAL!